BioCro/DARPA/upload_params.Rmd

```diff
@@ -177,3 +177,55 @@ Save cuticular conductance values as a .csv.
 write.csv(upload_cc, file = "upload_params_data/cut_cond.csv", row.names = FALSE)
 ```
 
+## Greenhouse physiological parameters Vmax, Rd, and stomatal slope
+
+Same as physiological parameters as above, but for greenhouse experiments
+
+Clean up data to have the final columns: 
+
+1. `notes`: records which row in the Google Sheet the record came from
+2. `local_datetime`: convert date into machine readable format
+3. `treatment`: specifies record's treatment using BETYdb treatment names
+4. `Vcmax`, `leaf_respiration_rate_m2`, `stomatal_slope.g1`, `stomatal_slope.BB`: measured values of physiological parameters specified
+5. `leafT`: leaf temperature, which is required by BETYdb
+
+```{r}
+upload_parameters_gr <- all_parameters %>% 
+  mutate(notes = paste("row", row_number(), "in https://docs.google.com/spreadsheets/d/1doQI0T7vav7LmNdMEoZJDx9iY_Imqa229CLRbRvurFc/#gid=967233489")) %>% 
+  filter(genotype == "ME034", 
+         TR %in% c("greenhouse"), 
+         trait %in% c("vmax", "Rd", "g1M", "g1BB")) %>% 
```

Comment:
I'm going to ask Tonantzin if there are any cuticular conductance (g0BB) values available for the greenhouse, so hold off on uploading this to the database until we either know there are none or we have those values available.